src/operator/ResourcesTable.tsx

```diff
@@ -0,0 +1,83 @@
+// Libraries
+import React, {FC, useContext} from 'react'
+import {
+  Table,
+  EmptyState,
+  ComponentSize,
+  Tabs,
+  Orientation,
+} from '@influxdata/clockface'
+
+// Components
+import ResourcesTableRow from 'src/operator/ResourcesTableRow'
+import ResourcesSearchbar from 'src/operator/ResourcesSearchbar'
+import {OperatorContext} from 'src/operator/context/operator'
+import OperatorTabs from 'src/operator/OperatorTabs'
+import PageSpinner from 'src/perf/components/PageSpinner'
+
+// Constants
+import {
+  accountHeaderInfo,
+  accountColumnInfo,
+  organizationColumnHeaders,
+  organizationColumnInfo,
+} from 'src/operator/constants'
+
+// Types
+import {Account, Organization} from 'src/types/operator'
+
+const ResourcesTable: FC = () => {
+  const {activeTab, accounts, organizations, status} = useContext(
+    OperatorContext
+  )
+
+  // TODO(ariel): remove type forcing here, this should be resolved with
+  // API integration
+  const resources: Account[] | Organization[] =
+    activeTab === 'accounts' ? accounts : organizations
+  const headers =
+    activeTab === 'accounts' ? accountHeaderInfo : organizationColumnHeaders
+  const infos =
+    activeTab === 'accounts' ? accountColumnInfo : organizationColumnInfo
```

Comment:
These are more conditionals that will have to change when we add the `Users` page back in.